osgar/lib/config.py

```diff
@@ -53,8 +54,14 @@ def config_load(*filenames, application=None):
 
 
 def merge_dict(dict1, dict2):
```

Comment:
The naming is somewhat confusing. It took me some time to realize that dict1 and dict2 and not necessarily dictionaries.

Reply:
I tried to refactor the code, maybe it is better now.